js/app/controllers/playercontroller.js

```diff
@@ -479,10 +493,61 @@ function ($scope, $rootScope, playlistService, Audio, gettextCatalog, Restangula
 		}
 	};
 
+	// Seekbar preview mouse support
+	$scope.seekbarPreview = function($event) {
+		if (!$scope.player.seekingSupported()) return;
+
+		var offsetX = $event.offsetX || $event.originalEvent.layerX;
+		var ratio = offsetX / $event.currentTarget.clientWidth;
+		var timestamp = ratio * $scope.position.total;
+		
+		$scope.position.currentPreview = timestamp;
+		$scope.position.currentPreview_ts = Date.now();
+	};
+
+	$scope.seekbarEnter = function() {
+		// Simple filter for transient mouse movements
+		$scope.position.currentPreview_tf = setTimeout(function() {
```

Comment:
This is unnecessarily convoluted. We can use the angular function $timeout, and the callback will be automatically executed within the angular digest loop. Something like
`$scope.position.currentPreview_tf = $timeout(() => $scope.position.currentPreview_tf = null, 100);`

Reply:
Unfortunately, this does not work. Sure, I can make use of the $timeout function, but the scope does not update on changed object properties. You have to manually apply the changes or set watchers. I have tried to find a better solution, but this was the only working one without setting watchers etc.

However, it's been a few years since I last used my AngularJS voodoo.. so don't take my word for it.

Reply:
I just tested with my suggested line of code and it worked perfectly for me.